### PR TITLE
Add reducer to my project

### PR DESCRIPTION
**Pull request by osbentech**
This is a React/redux project named bookStore App.
It will enable user to add, delete and edit books upon completion, for now it is a work in progress
below are few functionalities I have implemented:
- Enable user to add desired book
- display input books to the UI
- the initial state of check status in the categories tab changes upon click

### CODE REVIEW

File: src/redux/configurestore.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import bookReduce from './books/books';
+import categoryReduce from './categories/categories';
+
+const rootReducer = combineReducers({
+  book: bookReduce,
+  categ: categoryReduce,
+});
```
Comment (t-yanick) [inline src/redux/configurestore.js:8]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) for this project, " `Combine both reducers into a root reducer by using configureStore function.` ", kindly use configureStore to combine your reducers.

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const STAT = 'bookStore/categories/STATUS';
+
+const initialState = 'status Unknown';
+
+export default function categoryReduce(state = initialState, action) {
+  switch (action.type) {
+    case STAT:
+      return 'still constructing';
```
Comment (t-yanick) [inline src/redux/categories/categories.js]: - [ ] According to the requirements of this project, the check status action should return " `Under construction` ", please kindly fix that.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @osbentech, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @osbentech  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**